src/components/ChatMessageHeader.tsx

```diff
@@ -0,0 +1,46 @@
+// import { useNavigate } from "react-router-dom";
+import placeholder from "../assets/placeholder.png";
+import backbutton from "../assets/backbtn.svg";
+import { useEffect, useState } from "react";
+
+export const ChatMessageHeader = () => {
```

Comment:
Let's clean up this file a bit: 
- Some unnecessary commented lines
- Remove `fetch` until we integrate with the backend in another ticket
- Let's pass in the `user` props to this component from App.tsx, to 'feed' the name and image for the header